tests/network/hotplug.go

```diff
@@ -89,12 +88,7 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Expect(createBridgeNetworkAttachmentDefinition(testsuite.GetTestNamespace(nil), networkName, linuxBridgeName)).To(Succeed())
```

Comment:
This is a bit off-topic for this change, but it will help me in the review if `networkName` will change to something like `nadName` or `netAttDefName`.